eva-lib/src/test/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepositoryTest.java

```diff
@@ -234,7 +235,7 @@ public void testRegionIsFoundWithMafGreaterThan() {
         regions.add(region);
         List<VariantEntityRepositoryFilter> filters = new FilterBuilder().withMaf(">0.125").build();
         List<String> exclude = new ArrayList<>();
-        testFiltersHelperRegion(regions, filters, exclude, 37);
+        testFiltersHelperRegion(regions, filters, exclude, 43);
```

Comment:
I've now updated the variant test data from the production database, and updated and manually checked all the queries through the command line.